scss/components/_latest.scss

```diff
@@ -0,0 +1,119 @@
+@import "headline";
+@import "folder";
+@import "./mixins/flex-mix.scss";
+@import "./src/fonts";
+@import "./blog";
+
+.latest-portfolio {
+  background: $color-primary;
+}
+
+.portfolio {
+  margin-left: 174px;
+}
+
+.latest-headline {
+  @extend %headline;
+  padding-top: 124px;
+  letter-spacing: 0.6px;
+}
+
+.latest-underline {
+  margin-top: 14px;
+}
+
+.latest-underline-headline {
+  @extend %underline-headline;
+  color: $black;
+}
+
+.big-city {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  padding-top: 99px;
+}
+
+.big-city-cell {
```

Comment:
It would be better to use scss nest features:
```
.big-city {
  ...
  
  &-cell {...}
  &-headline {
    ...

    &-name {...}
  }

  ...
}
```

It related to all css.